app.js

```diff
@@ -0,0 +1,12 @@
+const menu = document.querySelector('.menu');
+const mobileNav = document.querySelector('.mobile-nav');
+
+menu.addEventListener('click', () => {
+  menu.classList.toggle('active');
+  mobileNav.classList.toggle('active');
+});
+
+document.querySelectorAll('menu-bar').forEach((n) => n.addEventListener('click', () => {
+  menu.classList.remove('active');
+  mobileNav.classList.remove('active');
+}));
```

Comment:
- [ ] When the user clicks (or taps) any of the mobile menu options, the mobile menu is supposed to disappear. So kindly fix that.

- [ ] When the user clicks (or taps) any of the mobile menu options, a correct part of the page is supposed to be displayed. So please fix that

![chrome-capture-2022-5-14](https://user-images.githubusercontent.com/68796232/173603876-be93bb68-669f-4f05-b0e9-541b3938f34f.gif)